app/src/forms/common/models/utils.js

```diff
@@ -0,0 +1,20 @@
+const toArray = (values) => {
+  if (values) {
+    return Array.isArray(values) ? values.filter(p => p && p.trim().length > 0) : [values].filter(p => p && p.trim().length > 0);
```

Comment:
`p.trim().length > 0` can be changed to just evaluating `p.trim().length` as they are equivalent.